apis/openstack-keystone/src/main/java/org/jclouds/openstack/keystone/v2_0/domain/PasswordCredentials.java

```diff
@@ -127,7 +127,7 @@ public boolean equals(Object obj) {
 
    protected ToStringHelper string() {
       return Objects.toStringHelper(this)
-            .add("username", username).add("password", password);
+            .add("username", username).add("password", "****");
```

Comment:
I would suggest to do `password == null ? null : password.replaceAll(".", "*")`
This will give us a hint in case of issues.